src/Illuminate/Mail/Transport/SesTransport.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Illuminate\Mail\Transport;
+
+use Aws\Exception\AwsException;
+use Aws\Ses\SesClient;
+use Exception;
+use Symfony\Component\Mailer\SentMessage;
+use Symfony\Component\Mailer\Transport\AbstractTransport;
+use Symfony\Component\Mailer\Transport\TransportInterface;
+
+class SesTransport extends AbstractTransport implements TransportInterface
+{
+    /**
+     * The Amazon SES instance.
+     *
+     * @var \Aws\Ses\SesClient
+     */
+    protected $ses;
+
+    /**
+     * The Amazon SES transmission options.
+     *
+     * @var array
+     */
+    protected $options = [];
+
+    /**
+     * Create a new SES transport instance.
+     *
+     * @param  \Aws\Ses\SesClient  $ses
+     * @param  array  $options
+     * @return void
+     */
+    public function __construct(SesClient $ses, $options = [])
+    {
+        $this->ses = $ses;
+        $this->options = $options;
+
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function doSend(SentMessage $message): void
+    {
+        try {
+            $this->ses->sendRawEmail(
+                array_merge(
+                    $this->options, [
+                        'Source' => $message->getEnvelope()->getSender()->toString(),
+                        'RawMessage' => [
+                            'Data' => $message->toString(),
+                        ],
+                    ]
+                )
+            );
+        } catch (AwsException $e) {
+            throw new Exception('Request to AWS SES API failed.', $e->getCode(), $e);
```

Comment:
```suggestion
            throw new Exception('Request to AWS SES API failed. ' . $e->getAwsErrorMessage(), $e->getCode(), $e);
```

When the develper have a try/catch wrapping this error, the final error message is unclear on what happened. By including AWS Error Message here the developer is able to identify what went wrong.
In my case, I was missing `ses:SendRawEmail` permission, but I was only able to detect that after modifying this line.

Reply:
@deleugpn can you maybe tackle that in a separate PR? Since this has always been the behaviour from 8.x and before. Just trying to get this working again. I do agree with you on that.

Reply:
I guess I will need to wait for this to get merged before asking to addres this because this file does not exist on the current stable branch so I can't propose a change yet